include/Note.h

```diff
@@ -237,10 +240,8 @@ class LMMS_EXPORT Note : public SerializingObject
 
 	static TimePos quantized( const TimePos & m, const int qGrid );
 
-	DetuningHelper * detuning() const
-	{
-		return m_detuning.get();
-	}
+	std::shared_ptr<DetuningHelper> detuning() const { return m_detuning; }
```

Comment:
```suggestion
	const std::shared_ptr<DetuningHelper>& detuning() const { return m_detuning; }
```
This could help prevent some unnecessary copies